Library/ViewModels/LoginToutViewModel.swift

```diff
@@ -208,6 +207,15 @@ public final class LoginToutViewModel: LoginToutViewModelType, LoginToutViewMode
     let appleSignInEvent = appleSignInInput
       .switchMap { input in
         AppEnvironment.current.apiService.signInWithApple(input: input)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .on(
+            starting: {
+              isLoading.value = true
+            },
+            failed: { _ in
```

Comment:
Notice that here we set the `isLoading` value to `false` only if the request fails, because we don't want to hide the processing view until the next request (`fetchUser`) terminates.